tests/vmi_multus_test.go

```diff
@@ -131,25 +131,26 @@ var _ = Describe("Multus", func() {
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 
-		// Create identical ptp crds in two different namespaces
+		// Create identical ptp crds
```

Comment:
This comment was for two following calls. Now you added a separate comment to the second one, so this does not make too much sense (to me) anymore. So this should be just "Create a PTP CRD"